app/client/scss/pages/_event-page.scss

```diff
@@ -0,0 +1,49 @@
+.event-page {
+  background-image: url(https://example.com/images/event-bg.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.event-page h1 {
```

Comment:
Instead of nesting like `.event-page h1 {`, just do:

```scss
.event-page {
  h1 {
    ...
  }
  ...
}
```

The same goes for all the selectors below.